src/ChooseSlot.js

```diff
@@ -0,0 +1,18 @@
+import moment from "moment";
```

Comment:
this package is not used and is not in `package.json` -> error

Reply:
Fixed